app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -28,11 +28,12 @@ const DeleteAlert = ({
   return (
     <Alert
       isOpen
+      className="delete-alert"
+      closeButton={false}
       isSubmitting={deleting}
       message="Are you sure you want to continue? This cannot be undone."
```

Comment:
`continue` with what? The alert messages should be explicit. Here we performing a specific irreversible action. Thus the messages should be updated accordingly.

Something like `continue deleting "${note.title}". This cannot be...` would be explicit.

Reply:
I understand that our message needs to be explicit and I will correct this at once. But I think it is not included in the Figma design. Also that raises a doubt. Do we, as developers, have the freedom to correct these designs or provide amendments.

Reply:
> Do we, as developers, have the freedom to correct these designs or provide amendments.

@navaneethsdk Most certainly. Feel free to make changes that are sensible and enhances the messages etc from the end user perspective. The Figma design is a base design. Everything in the Figma design ought to be added to your app. But enhancing or correcting mistakes is definitely your call and you should be doing the same.

Reply:
Ok. Thank you for the info, Yedhin.